src/target.rs

```diff
@@ -78,15 +78,27 @@ impl Target {
             || os == "haiku"
```

Comment:
Some lines above, why do we never set a version in the soname for Android @lu-zero ?

Reply:
because android does not have the version it its libraries.

Reply:
First time I hear that :) Do you have some more details about that?

Reply:
It is a well known annoyance, you may look at stackoverflow or the hopefully now closed bugs at common software such openssl ^^; I cannot find a document that explicitly mentions it but that's how it is, assuming nothing changed in the past year.

Reply:
How strange :) Thanks, I learned something new.

Reply:
@lu-zero: This is still true with the last NDKs. 
does not have and does not want !